src/components/sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,64 @@
+const Sidebar = () => {
+    return (
+        <aside className="px-4 py-1 my-2">
+            <div className="aside-heading">
+                <h5>Filters</h5>
+                <button className="button-link">Clear</button>
+            </div>
+            <hr />
+            <h6 className="my-1">Category</h6>
+            <ul className="side-nav font-medium pl-1">
+                <li>
+                    <input id="category-1" type="checkbox" />
+                    <label htmlFor="category-1">T-shirt</label>
+                </li>
+                <li>
+                    <input id="category-2" type="checkbox" />
+                    <label htmlFor="category-2">Hoodie</label>
+                </li>
+                <li>
+                    <input id="category-3" type="checkbox" />
+                    <label htmlFor="category-3">Cap</label>
+                </li>
+                <li>
+                    <input id="category-4" type="checkbox" />
```

Comment:
suggestion: see if you need the id here

Reply:
I am using id to bind the labels with the corresponding inputs